planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -509,8 +510,41 @@ bool NormalLaneChange::getLaneChangePaths(
 
 std::vector<DrivableLanes> NormalLaneChange::getDrivableLanes() const
 {
+  const auto & p = planner_data_->parameters;
+
+  lanelet::ConstLanelets extend_current_lanes{};
+  {
+    const auto lanelet_sequence = getRouteHandler()->getLaneletSequence(
+      status_.current_lanes.front(), 0.0, std::numeric_limits<double>::max());
+
+    lanelet::ConstLanelet closest_lane{};
+    if (!lanelet::utils::query::getClosestLaneletWithConstrains(
+          lanelet_sequence, getEgoPose(), &closest_lane, p.ego_nearest_dist_threshold,
+          p.ego_nearest_yaw_threshold)) {
+      return {};
+    }
+
+    extend_current_lanes = getRouteHandler()->getLaneletSequence(
+      closest_lane, p.backward_path_length, p.forward_path_length);
+  }
+
+  lanelet::ConstLanelets extend_target_lanes{};
+  {
+    const auto lanelet_sequence = getRouteHandler()->getLaneletSequence(
+      status_.lane_change_lanes.front(), 0.0, std::numeric_limits<double>::max());
+
+    lanelet::ConstLanelet closest_lane{};
+    if (!lanelet::utils::query::getClosestLaneletWithConstrains(
+          lanelet_sequence, getEgoPose(), &closest_lane, p.ego_nearest_dist_threshold,
+          p.ego_nearest_yaw_threshold)) {
+      return {};
+    }
+
+    extend_target_lanes = getRouteHandler()->getLaneletSequence(
+      closest_lane, p.backward_path_length, p.forward_path_length);
+  }
```

Comment:
```suggestion
    const auto extend_target_lanes = std::invoke([&]() -> lanelet::ConstLanelets {
      const auto lanelet_sequence = getRouteHandler()->getLaneletSequence(
        status_.lane_change_lanes.front(), 0.0, std::numeric_limits<double>::max());

      lanelet::ConstLanelet closest_lane{};
      if (!lanelet::utils::query::getClosestLaneletWithConstrains(
            lanelet_sequence, getEgoPose(), &closest_lane, p.ego_nearest_dist_threshold,
            p.ego_nearest_yaw_threshold)) {
        return {};
      }

      return getRouteHandler()->getLaneletSequence(
        closest_lane, p.backward_path_length, p.forward_path_length);
    });
```
The reason is similar to above.